src/BizHawk.Emulation.Common/Base Implementations/ControllerDefinition.cs

```diff
@@ -144,6 +155,11 @@ public static int PlayerNumber(string buttonName)
 
 		private static readonly Regex PlayerRegex = new Regex("^P(\\d+) ");
 
+		/// <summary>
+		/// Returns the number of players.
+		/// Currently only returns 0 for consoles where all control ports are empty,
+		/// so returns 1 for (non-linked) Game Boy and similar cases.
+		/// </summary>
 		public int PlayerCount

```

Comment:
Where is this used? Would it not be better to change the semantics such that player 0 counts as a player (that is, `PlayerCount` is usually 1 more than the number of plugged-in gamepads)?

Reply:
I'll look and consider it. I'm not sure I entirely like the semantics of treating the console itself as a "player" though. At that point, I would also question why one couldn't just get the length of `ControlsOrdered` directly.